portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,27 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+
+
+function addFavoriteShows() {
+  const shows =
+      ['Chuck', 'Lucifer', 'The Good Doctor', 'Love Death + Robots'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const show = shows[Math.floor(Math.random() * shows.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const showContainer = document.getElementById('show-container');
```

Comment:
I think the show-container isn't in the HTML yet (unless I missed it). Are you adding that in a follow-up?